eth/executionclient/execution_client.go

```diff
@@ -250,7 +262,9 @@ func (ec *ExecutionClient) Healthy(ctx context.Context) error {
 	if sp != nil {
 		ec.logger.Error("Execution client is not synced")
 		ec.metrics.ExecutionClientSyncing()
-		return fmt.Errorf("syncing")
+		if sp.CurrentBlock < sp.HighestBlock-ec.syncTolerance {
```

Comment:
in local testnets, this could result in overflow: `sp.HighestBlock-ec.syncTolerance`

it's better to do something like:
```
sp.HighestBlock > ec.syncTolerance && sp.CurrentBlock < sp.HighestBlock-ec.syncTolerance
```